src/SFA.DAS.ApplyService.Application/Apply/Financial/ClarificationFileUploadRequest.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using MediatR;
+
+namespace SFA.DAS.ApplyService.Application.Apply.Financial
+{
+    public class ClarificationFileUploadRequest : IRequest<bool>
```

Comment:
as mentioned above, think about the naming as you'll need to do a similar thing for deleting Clarification Files, so may be best to prefix this one with Add or write a generic version